src/cmd/flash.js

```diff
@@ -97,25 +96,35 @@ class FlashCommand {
 				}
 
 				switch (info.prefixInfo.moduleFunction) {
-					case MONOLITHIC:
+					case ModuleInfo.Function.MONO_FIRMWARE:
 						// only override if modular capable
 						destSegment = specs.systemFirmwareOne ? 'systemFirmwareOne' : destSegment;
 						break;
-					case SYSTEM_MODULE:
+					case ModuleInfo.Function.SYSTEM_PART:
 						destSegment = systemModuleIndexToString[info.prefixInfo.moduleIndex];
 						destAddress = '0x0' + info.prefixInfo.moduleStartAddy;
 						break;
-					case APPLICATION_MODULE:
+					case ModuleInfo.Function.USER_PART:
 						// use existing destSegment for userFirmware/factoryReset
 						break;
+					case ModuleInfo.Function.RADIO_STACK:
+						destSegment = 'radioStack';
+						destAddress = '0x0' + info.prefixInfo.moduleStartAddy;
+						break;
 					default:
 						if (!force) {
 							throw new Error('unknown module function ' + info.prefixInfo.moduleFunction + ', use --force to override');
 						}
 						break;
 				}
+
+				if (info.prefixInfo.moduleFlags & ModuleInfo.Flags.DROP_MODULE_INFO) {
+					return this._dropModuleInfo(binary);
+				}
+
+				return binary;
 			});
-		}).then(() => {
+		}).then((binary) => {
```

Comment:
There is already a local variable `binary` so this one shadows the earlier one. Can you rename this one to `finalBinary`?